composer.json

```diff
@@ -89,7 +89,7 @@
         "symfony/css-selector": "^4.4 || ^5.3 || ^6.0",
         "symfony/filesystem": "^4.4 || ^5.3 || ^6.0",
         "symfony/maker-bundle": "^1.25",
-        "symfony/phpunit-bridge": "^5.3 || ^6.0",
+        "symfony/phpunit-bridge": "^6.0",
```

Comment:
I think there are some new versions of symfony contracts, can you check if composer outdated is clean?

Reply:
Done